### PR TITLE
Mostafa

### PR DESCRIPTION
**Pull request by MostafaMoradii**

### CODE REVIEW

**Comment by MostafaMoradii** [comment]
search box added to topbar.

File: resources/views/partials/topbar.twig [hunk 1]
```diff
@@ -55,6 +38,37 @@
                                 </li>
                             </ul>
                         </li>
+                        {% set helpColor = setting_value('visiosoft.theme.defaultadmin::primary_color') %}
+                        {% if setting_value('visiosoft.theme.defaultadmin::dark_mode') %}
+                            {% set helpColor = setting_value('visiosoft.theme.defaultadmin::dark_text_color') %}
+                        {% endif %}
+                        <li class="nav-item dropdown floatright" style="padding-top: 5px;">
+                            <a href="javascript:void(0);" data-toggle="dropdown">
+                                <svg xmlns="http://www.w3.org/2000/svg" width="31.676" height="31.676" viewBox="0 0 31.676 31.676">
```
Comment (muammertopcu) [inline resources/views/partials/topbar.twig]: change to img() function for svg file

**Review by muammertopcu — CHANGES_REQUESTED** [review]

File: resources/views/partials/topbar.twig (hunk 1 above)
Comment (MostafaMoradii) [reply resources/views/partials/topbar.twig]: no, I need to change color in SVG dynamically. I defined a variable for color based on theme mode (light or dark), it used in the SVG

Comment (muammertopcu) [reply resources/views/partials/topbar.twig]: You can call it with the `{{ img('path').data|raw }} ` and you can replace it color with css

Comment (MostafaMoradii) [reply resources/views/partials/topbar.twig]: it is a good idea. thank you 🥇

**Review by muammertopcu — APPROVED** [review]

**Review by spektra2147 — APPROVED** [review]

**Pull request merged**